base/range.jl

```diff
@@ -66,8 +68,10 @@ Mathematically a range is uniquely determined by any three of `start`, `step`, `
 Valid invocations of range are:
 * Call `range` with any three of `start`, `step`, `stop`, `length`.
 * Call `range` with two of `start`, `stop`, `length`. In this case `step` will be assumed
-  to be one. If both arguments are Integers, a [`UnitRange`](@ref) will be returned.
-* Call `range` with one of `stop` or `length`. `start` and `step` will be assumed to be one.
+  to be positive one. If both arguments are Integers, a [`UnitRange`](@ref) will be returned.
+* Call `range` with one of `stop` or `length`. `start` and `step` will be assumed to be positive one.
+
+To construct a reverse range, specify a negative step size.
```

Comment:
```suggestion
To construct a reverse-order range, specify a negative step size, e.g. `5:-1:1 == [5,4,3,2,1]`.  Otherwise,
the default step of `+1` means that a range like `5:1` indicates an empty range (which is normalized to `5:4`,
so that the end of the range is one before the starting element).
```

Reply:
(This seems somewhat redundant with the earlier comment in the docstring — maybe delete the former?)

Reply:
I'm not sure which earlier comment you mean. Are you referring to the `:` docstring?